.drone.star

```diff
@@ -1105,7 +1105,7 @@ def smokeTests(ctx):
             "OCIS": "true",
         },
         "commands": [
-            "sleep 10 && yarn test:smoke:experimental tests/smoke/features/",
+            "sleep 10 && yarn test:smoke:experimental tests/smoke/features/ --retry 1",
```

Comment:
Please don't introduce retry here, the experience in the acceptance tests with it is bad enough. We should get rock-solid smoke tests instead of hiding flakyness

Reply:
@ScharfViktor could you remove the retries from both commands?